superset/assets/src/explore/components/controls/SubscriberLayer.jsx

```diff
@@ -110,23 +96,22 @@ export default class SubscriberLayer extends React.PureComponent {
       publishedSliceColumns,
       isNew: !this.props.name,
       validationErrors: {},
-      useAsModal,
-      actionType,
+      useAsModal
     };
 
     this.state.subscriptionList = this.state.isNew ? [{ columnType: '', operatorType: '', actionType: '', index: 0 }] : this.state.subscriptionList;
 
     this.handleSliceType = this.handleSliceType.bind(this);
+    this.handleColumnType = this.handleColumnType.bind(this);
+    this.handleOperatorType = this.handleOperatorType.bind(this);
     this.submitSubscription = this.submitSubscription.bind(this);
```

Comment:
What is the difference between submitSubscription and applySubscription?

Reply:
submitSubscription will applySubscription and close the popup, where as applySubscription will apply selected subscription